app/src/common/urls.js

```diff
@@ -195,6 +195,10 @@ export const URLS = {
   statisticsServerSettings: () => `${urlBase}settings/analytics`,
   events: (projectId) => `${urlBase}${projectId}/activity`,
   allUsers: () => `${urlBase}user/all`,
+  searchUsers: (term) =>
+    `${urlBase}user/all${getQueryParams({
+      'filter.cnt.email': term,
```

Comment:
We need to search by email and login. Maybe there is a special filter for this or a way to combine two filters.